samples/bookinfo/build_push_update_images.sh

```diff
@@ -57,7 +61,7 @@ do
 done
 
 #Build docker images
-src/build-services.sh "${VERSION}" "${PREFIX}"
+ENABLE_MULTIARCH_IMAGES="${ENABLE_MULTIARCH_IMAGES}" src/build-services.sh "${VERSION}" "${PREFIX}"
```

Comment:
I suggest passing parameters，just like "${VERSION}" "${PREFIX}"

Reply:
prefer keeping it this way so ENABLE_MULTIARCH_IMAGES does not become a mandatory parameter for `build-services.sh`